pkgs/development/libraries/vectorscan/default.nix

```diff
@@ -0,0 +1,62 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, ragel
+, util-linux
+, python3
+, boost
+, enableShared ? !stdenv.hostPlatform.isStatic
+}:
+
+stdenv.mkDerivation rec {
+  pname = "vectorscan";
+  version = "5.4.9";
+
+  src = fetchFromGitHub {
+    owner = "VectorCamp";
+    repo = "vectorscan";
+    rev = "vectorscan/${version}";
+    hash = "sha256-V5Qgr8aH1H+ZoJ0IZ52HIDRZq+yIwHjLf3gU/ZhjjlY=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    ragel
+    util-linux
+    python3
+  ];
+
+  buildInputs = [
+    boost
+  ];
+
+  cmakeFlags = lib.optional (enableShared) [
```

Comment:
```suggestion
  cmakeFlags = lib.optionals enableShared [
```

Reply:
after reorganizing i opted to use lib.optional for single options